pkg/exchange/okex/parse.go

```diff
@@ -363,3 +373,59 @@ func parseOrder(v *fastjson.Value) ([]okexapi.OrderDetails, error) {
 
 	return orderDetails, nil
 }
+
+func toGlobalSideType(side okexapi.SideType) (types.SideType, error) {
+	switch side {
+	case okexapi.SideTypeBuy:
+		return types.SideTypeBuy, nil
+
+	case okexapi.SideTypeSell:
+		return types.SideTypeSell, nil
+
+	default:
+		return types.SideType(side), fmt.Errorf("unexpected side: %s", side)
+	}
+}
+
+type MarketTradeEvent struct {
+	InstId    string                     `json:"instId"`
+	TradeId   string                     `json:"tradeId"`
```

Comment:
you can use StrInt here?

Reply:
see pkg/types/strint.go

Reply:
or maybe you can create a StrUint for this?

Reply:
I asked for official support, which confirmed that the trade ID will be numeric. Therefore, I made the necessary changes to fix it.